app/views/student/show.html.erb

```diff
@@ -8,83 +23,88 @@
+      <div style="">
+        <% @student = Student.find(params[:id]) %>
+        <div class="list-group">
+          <li class="list-group-item"><strong>Email </strong><span class="pull-right"> <%= @student.email %> </span></li>
+          <li class="list-group-item"><strong>Roll Year </strong><span class="pull-right"> <%= @student.rollnum %> </span></li>
+          <li class="list-group-item"><strong>Join Year </strong><span class="pull-right"> <%= @student.joinyear %> </span></li>
+          <li class="list-group-item"><strong>Department </strong><span class="pull-right"> <%= @student.department %> </span></li>
+          <li class="list-group-item"><strong>Hall </strong><span class="pull-right"> <%= @student.hall %> </span></li>
+        </div>
+        <p> <a style="color:black;border-color:black" href="#" class="btn btn-info" role="button">Edit your Profile</a></p>
       </div>
-      <p> <a style="color:black;border-color:black" href="#" class="btn btn-info" role="button">Edit your Profile</a></p>
-    </div>
-  </div><!--col-md-3 ends -->
-
+    </div><!--col-md-3 ends -->
 
-  <div class="col-md-6">
 
-      <!--
+    <div class="col-md-6">
      <ul class="nav nav-pills nav-justified" role="tablist">
         <li role="presentation" class="active"><a href="#" style="color:black">Home<span class="badge pull-right">42</span></a></li>
         <li role="presentation"><a href="#" style="color:black">My Messages &nbsp;<span class="label label-primary"> 3 new </span></a></li>
      </ul>
-   -->
-   <div style="margin:30px 0px 5px 0px" class="well">
-    <% if @mentorallotted %>
-    <h3>Feel Free to get in touch with your Mentor</h3>
-    <a href="mailto: <%= @mentor.email %>">
-      <h3>Click here</h3>
-    </a>
-    <% else %>
-    <h3>We will allot a mentor to you soon.</h3>
-    <% end %>
-  </a>
-</div>
-
-
-<div class="row" style="margin:5px 4px 5px 4px">
- <h1>Recent updates </h1>
- <div class="well">
-  <img src="/assets/images/banner.jpg" alt="..." height="400px" width="500px;">
-  <div class="caption">
-    <h3>Thumbnail label</h3>
-    <p>...</p>
-    <p><a href="#" class="btn btn-primary" role="button">Read More</a></p>
-  </div>
-</div>
+     <div style="margin:30px 0px 5px 0px" class="well">
+       <h3 >Feel Free to get in touch with your Mentor</h3>
+       <div class="alert alert-danger">Messaging Service is currently down for Maintenance</div>
+       <!-- <div class="form-group">
+          <input type="text" class="form-control " placeholder="Subject">
+       </div>
+       <div class="form-group">
+          <textarea class="form-control " placeholder="Your Message" style="height:100px"></textarea>
+       </div>
+
+       <a href="#" class="btn btn-primary" id="send" role="button" style="border-color:black ">SEND</a> -->
+     </div>
+    
+
+      <div class="row" style="margin:5px 4px 5px 4px">
+        <h1>Recent updates </h1>
+        <div class="well">
+          <img src="det_pic.jpg" alt="...">
```

Comment:
Please refactor the path of the images in this file. 

All the images must have a `src` attribute from `/assets/images/imagename.jpg`